src/main/java/frc/robot/subsystems/PCM.java

```diff
@@ -14,7 +14,7 @@ public class PCM extends SubsystemBase {
   public final Compressor compressor;
   /** Creates a new PCM. */
   public PCM() {
-    compressor = new Compressor(PCM_CAN_ID, PneumaticsModuleType.REVPH);
+    compressor = new Compressor(PCM_CAN_ID, PneumaticsModuleType.CTREPCM);
```

Comment:
hey harlen - what is this?

Reply:
I changed the pneumatics type of the compressor after learning what type we used in the robot.